descwl_shear_sims/sim/dmpsfs.py

```diff
@@ -0,0 +1,179 @@
+import galsim
```

Comment:
The class hierarchy in this file is mangled a bit. The shared methods in the fixed PSF and the power spectrum ones should be put into a new base class that inherits from ImagePsf. Then those two objects should inherit from that new base class.